installation/sql/postgresql/base.sql

```diff
@@ -258,7 +259,8 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'mod_privacy_dashboard', 'module', 'mod_privacy_dashboard', '', 1, 1, 1, 0, 1, '', '{}', '', 0, 0),
 (0, 'mod_submenu', 'module', 'mod_submenu', '', 1, 1, 1, 0, 1, '', '{}', '', 0, 0),
 (0, 'mod_privacy_status', 'module', 'mod_privacy_status', '', 1, 1, 1, 0, 1, '', '{}', '', 0, 0),
-(0, 'mod_guidedtours', 'module', 'mod_guidedtours', '', 1, 1, 1, 0, 1, '', '{}', '', 0, 0);
+(0, 'mod_guidedtours', 'module', 'mod_guidedtours', '', 1, 1, 1, 0, 1, '', '{}', '', 0, 0),
+(0, 'mod_community_info', 'module', 'mod_community_info', '', 1, 1, 1, 0, 1, '', '{}', '');
```

Comment:
```suggestion
(0, 'mod_community_info', 'module', 'mod_community_info', '', 1, 1, 1, 0, 1, '', '{}', '', 0, 0);
```
This should fix the `42601, 7, ERROR: VALUES lists must all be the same length LINE 28: (0, 'mod_community_info', 'module', 'mod_community_info', ''... ` error mentioned by @muhme in his comment https://github.com/joomla/joomla-cms/pull/43753#issuecomment-2323311686 . The base.sql for PostgreSQL differs here compared to the one for MySQL for historical reasons. It specifies values for ordering and state. That's not really necessary as these columns have the right default values, but it would be out of scope of this PR here to change that.

Reply:
You can apply this suggestion on GitHub with a button. After that, Drone should pass. If it doesn't I will check again.

Reply:
@richard67 Is there any other problem with the base.sql for PostgreSQL?

Reply:
Yes, see https://github.com/joomla/joomla-cms/pull/43753#discussion_r1769562995 .